src/sdk-reference/js/6/profile/getRoles/snippets/getRoles.js

```diff
@@ -0,0 +1,29 @@
+const credentials = { username: 'foo', password: 'bar' };
+
+try {
+  await kuzzle.auth.login('local', credentials);
+
+  const user = await kuzzle.auth.getCurrentUser();
+
+  const profiles = await user.getProfiles();
+
+  const roles = await profiles[0].getRoles();
+  console.log(roles);
+  /*
+    [
+      Role {
+        _id: 'default',
+        controllers: {
+          '*': {
+            actions: {
+              '*': true
+            }
+          }
+        }
+      }
+    ]
+  */
+  console.log(`Successfully retrieved ${roles.length} roles.`);
```

Comment:
This console.log should be removed, and the previous one should be tested instead: it's clearer that way (less code to read for users), and this makes the test more fiable.

```suggestion
```